src/main/java/software/amazon/awssdk/aws/greengrass/GeneratedAbstractUpdateThingShadowOperationHandler.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Should be possible to add these classes only in the shadow service itself.

Reply:
We need a reference to this in the `GreengrassCoreIPCService` when setting the handlers, can see it [here](https://github.com/aws-greengrass/aws-greengrass-nucleus/blob/local-shadow-service-update/src/main/java/software/amazon/awssdk/aws/greengrass/GreengrassCoreIPCService.java#L228-L246).

Reply:
could we use the `[setOperationHandler](https://github.com/aws-greengrass/aws-greengrass-nucleus/blob/local-shadow-service-update/src/main/java/software/amazon/awssdk/aws/greengrass/GreengrassCoreIPCService.java#L264)` method - but call it from within the shadow manager?

Reply:
Yes we can do that, we should be able to move these generated abstract classes to the ShadowManager and move the `set<Operation>Handlers` to the ShadowManager.

Reply:
This PR was updated to reflect this, also relevant [PR](https://github.com/aws-greengrass/aws-greengrass-shadow-manager/pull/42) made to ShadowManager to support this change as well.